src/AspNetCoreModuleV2/OutOfProcessRequestHandler/processmanager.cpp

```diff
@@ -200,22 +140,14 @@ PROCESS_MANAGER::GetProcess(
                 ASPNETCORE_EVENT_RAPID_FAIL_COUNT_EXCEEDED_MSG,
                 pConfig->QueryRapidFailsPerMinute());
 
-            hr = HRESULT_FROM_WIN32(ERROR_SERVER_DISABLED);
-            goto Finished;
+            RETURN_IF_FAILED(HRESULT_FROM_WIN32(ERROR_SERVER_DISABLED));
```

Comment:
Can we make a macro that is simply `RETURN` rather than `RETURN_IF_FAILED`?

Reply:
Yep